### DISCUSSION TITLE
7.19 일일 회의

### DISCUSSION THREAD

**Discussion by jlee38266** [3sam5oh/webtoon-recommendation-service#61] [일일 회의]
## Iteration1 회고
![image](https://github.com/user-attachments/assets/20c74353-778d-4bec-a9c4-8db0d334d94f)

![image](https://github.com/user-attachments/assets/477fee07-7c58-4ef3-b3c9-226f2e5e248d)

**Comment by koyoung99** [comment]
- 헥사고널아키텍처 열심히열심히 이해하기
- naver oauth 헥사고널아키텍처 구조로 바꾸기

이거면 주말 후딱일거같아요 ~

**Comment by t4e1** [comment]
- spring boot 커스텀 메트릭 구현(1순위)
- spring boot 로깅 생각해보기(2순위)
- local 테스트 환경 구현해보기(가능하다면)

**Comment by jlee38266** [comment]
- LocalStack 튜토리얼 학습
- terraform으로 LocalStack 활용 연습
